src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.taskmanagement.TaskManager;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        String userInput = "";
+        TaskManager taskList = new TaskManager();
+        printGreeting();
+        while (!userInput.equals("bye")) {
+            printLine();
+            userInput = in.nextLine().trim();
+            String[] words = userInput.split(" ");
+            String command = words[0];
```

Comment:
You've done a good job with the abstracting out the methods for cases below. How about ensuring single level of abstraction here?